homeassistant/components/avri/sensor.py

```diff
@@ -0,0 +1,100 @@
+"""Support for Avri waste curbside collection pickup."""
+import logging
+
+from avri.api import Avri, AvriException
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+CONF_COUNTRY_CODE = "country_code"
+CONF_POSTCODE = "postcode"
+CONF_HOUSE_NUMBER = "house_number"
+CONF_HOUSE_NUMBER_EXTENSION = "house_number_extension"
+DEFAULT_NAME = "avri"
+ICON = "mdi:trash-can-outline"
+SCAN_INTERVAL = 86400
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_POSTCODE): cv.string,
+        vol.Required(CONF_HOUSE_NUMBER): cv.string,
+        vol.Optional(CONF_HOUSE_NUMBER_EXTENSION, default=""): cv.string,
```

Comment:
Please try to avoid empty strings as default values. Does this house number extension need a default value at all?

Reply:
Fixed it by moving it to the constructor.